man-roxygen/examples-class-ordered.R

```diff
@@ -0,0 +1,43 @@
+#' @examples
+#' # Two class
+#' data("two_class_example")
+#' two_class_example$truth <- factor(
+#'   two_class_example$truth, levels(two_class_example$truth), ordered = TRUE
+#' )
+#' two_class_example$predicted <- factor(
+#'   two_class_example$predicted, levels(two_class_example$truth), ordered = TRUE
+#' )
+#' <%=metric_fn %>(two_class_example, truth, predicted)
+#'
+#' # Multiclass
+#' library(dplyr)
+#' data(hpc_cv)
+#' hpc_cv$obs <- factor(
+#'   hpc_cv$obs, levels(hpc_cv$obs), ordered = TRUE
+#' )
+#' hpc_cv$pred <- factor(
+#'   hpc_cv$pred, levels(hpc_cv$obs), ordered = TRUE
+#' )
+#'
+#' hpc_cv %>%
+#'   filter(Resample == "Fold01") %>%
+#'   <%=metric_fn %>(obs, pred)
+#'
+#' # Groups are respected
+#' hpc_cv %>%
+#'   group_by(Resample) %>%
+#'   <%=metric_fn %>(obs, pred)
+#'
+#' # Weighted macro averaging
+#' hpc_cv %>%
+#'   group_by(Resample) %>%
+#'   <%=metric_fn %>(obs, pred, estimator = "macro_weighted")
```

Comment:
If it extends naturally to multiclass, does this do anything?

Reply:
I just copied from examples-class, just changing it to be ordered. I tried not to change too much for fear that it might blow up. Note that kap.Rd (which also extends naturally to multiclass) has the same extraneous "Weighted macro averaging" block.